web/src/features/prompts/components/NewPromptForm/index.tsx

```diff
@@ -102,7 +102,14 @@ export const NewPromptForm: React.FC<NewPromptFormProps> = (props) => {
     onError: (error) => setFormError(error.message),
   });
 
-  const allPrompts = api.prompts.all.useQuery({ projectId }).data;
+  const allPrompts = api.prompts.all.useQuery({
```

Comment:
need to use options here now, otherwise this will not include all names as we introduce pagination